package.json

```diff
@@ -25,7 +25,15 @@
     "url": "https://github.com/blackbaud/skyux-sample/issues"
   },
   "homepage": "https://github.com/blackbaud/skyux-sample#readme",
-  "peerDependencies": {},
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/core": "^3.1.0",
+    "@skyux/i18n": "^3.0.1",
+    "@skyux/popovers": "^3.0.0-rc.0",
```

Comment:
When typing `npm install` I see: 
"@skyux/popovers@3.0.0-rc.0 requires a peer of @skyux/indicators@>=3.0.0-rc.0 but none is installed. You must install peer dependencies yourself."
We should probably add indicators to this module's dev dependencies.

Reply:
done